src/app/services/manage-people/manage-people.service.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { ManagePeopleService } from './manage-people.service';
+
+describe('ManagePeopleService', () => {
+  beforeEach(() => TestBed.configureTestingModule({}));
+
+  it('should be created', () => {
+    const service: ManagePeopleService = TestBed.get(ManagePeopleService);
```

Comment:
Add blank line here

Reply:
`let service: AgeVerificationService;` should be declared after `describe`. You will reuse this var for another tests.